main.py

```diff
@@ -32,16 +40,63 @@ def has_required_role(ctx):
     role = discord.utils.get(ctx.guild.roles, id=dev_role_id)
     return role in ctx.author.roles
 
+def send_coins(rpc_user, rpc_password, rpc_port, rpc_ip, from_address, to_address, amount):
+    rpc_url = f'http://{rpc_user}:{rpc_password}@{rpc_ip}:{rpc_port}'
+    headers = {'content-type': 'application/json'}
+
+    def send_coins(from_address, to_address, amount):
+        payload = {
+            "method": "sendtoaddress",
+            "params": [to_address, amount],
+            "jsonrpc": "2.0",
+            "id": "1"
+        }
+        response = requests.post(rpc_url, data=json.dumps(payload), headers=headers).json()
+        return response
+
+    response = send_coins(from_address, to_address, amount)
+    if response.get('error') is not None:
+        print("Error:", response['error'])
+
+    return response
+
+def get_balance(username, password, host, port, address):
+    url = f'http://{host}:{port}'
+    auth = HTTPBasicAuth(username, password)
+    data = {
+        "jsonrpc": "2.0",
+        "id": 0,
+        "method": "getbalance",
+        "params": []
+    }
+    response = requests.post(url, json=data, headers={'Content-Type': 'application/json'}, auth=auth)
+    if response.status_code != 200:
+        raise Exception(f"HTTP error: {response.json()}")
+    result = response.json()
+    if 'error' in result and result['error'] is not None:
+        raise Exception(result['error'])
+    return result['result']
+
+
 @bot.command(name='verify')
 async def verify_wallet(ctx, address: str):
-    async with aiohttp.ClientSession() as session:
-        async with session.get(f"https://explorer.cmusic.ai/ext/getaddress/{address}") as response:
-            if response.status == 200:
-                data = await response.json()
-                json_data = json.dumps(data, indent=2)
-                await ctx.send("```json\n" + json_data + "```")
-            else:
-                await ctx.send("Failed to fetch address information.")
+    if pattern.match(address):
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"https://explorer.cmusic.ai/ext/getaddress/{address}") as response:
+                if response.status == 200:
+                    data = await response.json()
+                    json_data = json.dumps(data, indent=2)
+                    await ctx.send("```json\n" + json_data + "```")
+                else:
+                    await ctx.send("Failed to fetch address information, wallet may not be valid or no transactions [failed explorer check].")
+    else:
+        await ctx.send("Wallet address does not seem valid, [failed regex check].")
+
+@bot.command(name='airdropbal')
+@commands.check(has_required_role)
+async def airdropbal(ctx):
+    balance_response = get_balance(rpc_user, rpc_password, rpc_ip, rpc_port, from_address)
+    await ctx.send("CMS Balance: " + str(balance_response))
```

Comment:
Can we also set the ticker in the environmental variables.  I'd like to open this up so anyone can use it with any coin.